OracleDatabase/12.1.0.2/Vagrantfile

```diff
@@ -20,38 +20,91 @@ VAGRANTFILE_API_VERSION = "2"
 BOX_URL = "https://oracle.github.io/vagrant-boxes/boxes"
 BOX_NAME = "oraclelinux/7"
 
-# define hostname
-NAME = "oracle-12102-vagrant"
-
 unless Vagrant.has_plugin?("vagrant-proxyconf")
   puts 'Installing vagrant-proxyconf Plugin...'
   system('vagrant plugin install vagrant-proxyconf')
 end
 
-# get host time zone for setting VM time zone, if possible
-# can override in env section below
-offset_sec = Time.now.gmt_offset
-if (offset_sec % (60 * 60)) == 0
-  offset_hr = ((offset_sec / 60) / 60)
-  timezone_suffix = offset_hr >= 0 ? "-#{offset_hr.to_s}" : "+#{(-offset_hr).to_s}"
-  SYSTEM_TIMEZONE = 'Etc/GMT' + timezone_suffix
-else
-  # if host time zone isn't an integer hour offset, fall back to UTC
-  SYSTEM_TIMEZONE = 'UTC'
+# Define constants
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  # Use vagrant-env plugin if available
+  if Vagrant.has_plugin?("vagrant-env")
+    config.env.load(".env.local", ".env") # enable the plugin
+  end
+
+  # VM name
+  NAME = default_s('NAME', 'oracle-12102-vagrant')
+
+  # Memory for the VM (in MB, 2048 MB = 2 GB)
+  MEMORY = default_i('MEMORY', 2048)
```

Comment:
There is the memory for the VM and there is the memory for the database.
We do not set the memory for the database today but just inherit what `dbca` picks by default (which is 40% of the existing memory on the environment).

`MEMORY` is probably too vague, does it mean VM memory or DB memory? We should consider using a more explicit variable, such as `VM_MEMORY` (I do see the irony of my next comment asking why we have the `VM_` prefix for other variables and am happy to hear other suggestions).

Reply:
`VM_MEMORY` makes sense to me. I'll change it.

Reply:
I've changed `MEMORY` to `VM_MEMORY` in all `.env` and `Vagrantfile` files and updated the PR.